src/api/depositV2.js

```diff
@@ -59,6 +59,9 @@ module.exports = async (dvf, data, nonce, signature) => {
   let transactionHashCb
   const transactionHashPromise = new Promise((resolve, reject) => {
     transactionHashCb = (err, result) => {
+      if (txHashCb) {
+        txHashCb()
```

Comment:
You might want to pass `result` as parameter (if no error is raised) since it could be useful in other contexts

Reply:
Added.